benchmarks/sorting.rb

```diff
@@ -3,7 +3,7 @@
 require 'benchmark'
 require 'daru'
 
-vector = Daru::Vector.new(10000.times.map.to_a.shuffle)
+vector = Daru::Vector.new(1000.times.map.to_a.shuffle)
 df = Daru::DataFrame.new({
```

Comment:
Let the benchmarking code be the same in the final commit. Also, just append your results at the end of the file without deleting anything so we can keep track of speed over time.